webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityHeadSpace.vue

```diff
@@ -39,8 +39,16 @@ export default {
     spaceId() {
       return this.space && this.space.id;
     },
+    testvisibility(){
```

Comment:
testVisibility is not a function name easy to read.
Can you change to something like 
canClickOnSpace

Reply:
ok I changed it